amplfi/train/models/flow.py

```diff
@@ -141,16 +150,40 @@ def validation_step(self, batch, _):
         )
         return loss
 
+    def on_predict_epoch_start(self):
+        self.on_test_epoch_start()
```

Comment:
what is the difference between predict_epoch and start_epoch in lightning lingo?

Reply:
See discussion below